host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1265,4 +1266,12 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @LogMessage(level = Level.ERROR)
     @Message( id = 172, value = "Failed executing the suspend operation for server: %s")
     void suspendExecutionFailed(@Cause IOException cause, String serverName);
+
+    @LogMessage(level = Level.INFO)
+    @Message(id = 173, value = "Timed out after %d ms awaiting server prepared response(s) for server: %s")
+    void timedOutAwaitingPreparedResponse(@Cause TimeoutException e, int blockingTimeout, String serverName);
```

Comment:
I suggest changing the name of this method and the one below and the method text to refer to "suspend" rather than "prepared".

When we use "prepared" in messages end users may see we only mean the initial response the server sends, which you read in ServerInventoryImpl L444. But you can get timeouts or interrupted exceptions both there and when waiting for the final response in L454. So in messages that may be used due to L454, we shouldn't refer to "prepared".